Core/HLE/sceKernelModule.cpp

```diff
@@ -1274,6 +1274,11 @@ static Module *__KernelLoadELFFromPtr(const u8 *ptr, size_t elfSize, u32 loadAdd
 		std::vector<SectionID> codeSections = reader.GetCodeSections();
 		for (SectionID id : codeSections) {
 			u32 start = reader.GetSectionAddr(id);
+			if (!Memory::IsValidAddress(start)) {
```

Comment:
Maybe should validate end too, clamping it to valid.  I wonder if this is some game hack.

-[Unknown]

Reply:
Could very well be, good to have the checks either way though. Yeah, I could have used IsValidRange, I'll switch to that.